### PR TITLE
New jupyterlab-dataops image

### PR DESCRIPTION
**Pull request by ddebowczyk92**

### CODE REVIEW

File: .github/workflows/release_dataops.yml [hunk 1]
```diff
@@ -0,0 +1,56 @@
+name: Build & Publish jupyterlab-dataops Image
```
Comment (szczeles) [inline .github/workflows/release_dataops.yml:1]: I think we can keep one `release.yml` file and just add the multiple tags for him to watch. What do you think?

**Review by szczeles — COMMENTED** [review]
I vote to not keep the extensions binaries in our repo and instead download them from the source: https://github.com/innoverio/vscode-dbt-power-user/releases

**Comment by ddebowczyk92** [comment]
> I vote to not keep the extensions binaries in our repo and instead download them from the source: https://github.com/innoverio/vscode-dbt-power-user/releases

installing extensions from the source would be quite an overhead because it requries compilation, instead extensions are downloaded straight from vs marketplace repo

**Comment by szczeles** [comment]
> > I vote to not keep the extensions binaries in our repo and instead download them from the source: https://github.com/innoverio/vscode-dbt-power-user/releases
> 
> installing extensions from the source would be quite an overhead because it requries compilation, instead extensions are downloaded straight from vs marketplace repo

Ah, got it.

File: .github/workflows/release_dataops.yml (hunk 1 above)
Comment (ddebowczyk92) [reply .github/workflows/release_dataops.yml:1]: but how can we define context (directory which should be built) based on tag?

Comment (szczeles) [reply .github/workflows/release_dataops.yml]: It's already defined in the lower part by extracting the directory name from the tag itself

File: README.md [hunk 2]
```diff
@@ -26,6 +26,25 @@ Compatibility:
...
+image with additional setup:
+
+* conda-backed python 3.8.8 environment
+* [jupyterlab-git](https://github.com/jupyterlab/jupyterlab-git) extension
+* [jupyter-server-proxy](https://jupyter-server-proxy.readthedocs.io) extension
+* local [code-server](https://github.com/coder/code-server) as IDE accessed from the browser with
+[dbt Power User](https://marketplace.visualstudio.com/items?itemName=innoverio.vscode-dbt-power-user) extension pre-installed
+* [dbt](https://docs.getdbt.com/) CLI
+
+![jupyterlab-dataops-launcher](docs/jupyterlab-dataops-launcher.png)
```
Comment (szczeles) [inline README.md:41]: You can drop the `img.png`, as it doesn't seem in use

**Review by szczeles — APPROVED** [review]
Looks great! Just drop the unused image and it's ready to merge

**Pull request merged**